### PR TITLE
Added an icon when the news was edited

### PR DESCRIPTION
**Pull request by toster234**
Improved to acceptable appearance than in https://github.com/Wruczek/ts-website/pull/218, need to improve date display in tooltip.

### CODE REVIEW

File: src/js/script.js [hunk 1]
```diff
@@ -105,6 +105,13 @@ $(function () {
             }
         });
     }
+    // News edited date
+    const newsEditedElements = document.querySelectorAll('.news-edited');
```
Comment (Wruczek) [inline src/js/script.js]: Please use jQuery (to be consistent with the rest of the code base).

File: src/private/templates/index.latte [hunk 2]
```diff
@@ -31,7 +31,12 @@
                             {$news["title"]}
                         {/ifset}
                     </h5>
-                    <div>{strtotime($news["added"])|fuzzyDateAbbr}</div>
+                    <div>
+                        {strtotime($news["added"])|fuzzyDateAbbr}
+                        {if $news["edited"] !== null}
+					        <i class="fas fa-pencil-alt mr-0 ml-1 news-edited" data-toggle="tooltip" n:attr="data-date => $news['edited']" data-html="true"></i>
```
Comment (Wruczek) [inline src/private/templates/index.latte]: 1. What's the purpose of the `news-edited` class?
2. Why is the HTML allowed in the tooltip?

File: src/js/script.js [hunk 3]
```diff
@@ -105,6 +105,13 @@ $(function () {
...
+    newsEditedElements.forEach(element => {
+    const editedDate = element.getAttribute('data-date');
+    const tooltipTitle = `${dayjs(editedDate).format('D MMMM YYYY MM:HH')}`;
```
Comment (Wruczek) [inline src/js/script.js]: Please use `timestampToDate(timestamp, true)` function to format time.

**Review by Wruczek — CHANGES_REQUESTED** [review]

File: src/private/templates/index.latte (hunk 2 above)
Comment (toster234) [reply src/private/templates/index.latte]: 1. The `news-edited` class is to make javascript recognize the element.
2. Deleted.

File: src/js/script.js (hunk 3 above)
Comment (toster234) [reply src/js/script.js]: Done.

**Comment by Justman100** [comment]
@Wruczek

**Review by Wruczek — APPROVED** [review]

**Pull request merged**